services/QuillLMS/client/app/bundles/Comprehension/components/studentView/container.tsx

```diff
@@ -374,8 +375,9 @@ export class StudentViewContainer extends React.Component<StudentViewContainerPr
     const { currentActivity, } = activities
     const { submittedResponses, } = session
     if (!currentActivity) return
-
-    const steps =  currentActivity.prompts.map((prompt, i) => {
+    // sort by conjunctions in alphabetical order: because, but, so
+    const filteredSteps = _.sortBy( currentActivity.prompts, 'conjunction' );
```

Comment:
My preference is always to use native methods where possible, so I would do `currentActivity.prompts.sort((a, b) => a.conjunction.localeCompare(b.conjunction))` here instead of including `underscore`.

Reply:
why do you prefer them? for compatibility reasons?